sql/weenies/PK Trophy Vendor/4200101 Turquoise Winged Helmet.sql

```diff
@@ -0,0 +1,46 @@
+DELETE FROM `weenie` WHERE `class_Id` = 4200101;
+
+INSERT INTO `weenie` (`class_Id`, `class_Name`, `type`, `last_Modified`)
+VALUES (4200101, 'helmetgromniewingedtailor', 2, '2005-02-09 10:00:00') /* Clothing */;
+
+INSERT INTO `weenie_properties_int` (`object_Id`, `type`, `value`)
+VALUES (4200101,   1,          2) /* ItemType - Armor */
+     , (4200101,   3,          1) /* PaletteTemplate - AquaBlue */
+     , (4200101,   4,          1) /* ClothingPriority - Head */
```

Comment:
clothing priority for head should be 16384

Reply:
submitted the change